defaults/main.yml

```diff
@@ -8,10 +8,21 @@ docker_apt_repo: "deb https://apt.dockerproject.org/repo ubuntu-{{ ansible_distr
 docker_yum_gpg_key_url:  'https://yum.dockerproject.org/gpg'
 docker_yum_repo_baseurl: 'https://yum.dockerproject.org/repo/main/centos/$releasever/'
 
+# Used when package name is not docker-engine
+docker_package: docker-engine
+docker_package_version:  '*'    # specify '*' for latest version
+
 docker_engine_version:  '*'    # specify '*' for latest version
```

Comment:
Pls consider replacing `docker_engine_version` with the new `docker_package_version` since they both relate to `docker_package`